lessonThree.py

```diff
@@ -10,50 +10,67 @@
 # If the password for that user doesn't match then it should return "password is incorrect"
 
 
-users = []
+users = ["Alayna", "Lance", "Grayson", "Vivian"]
 
 def createUser(fullName, email, userName, password, phoneNumber) :
     # your code here
+    user = []
+    
+    for user in user :
+      user.append({
+        'user': 'user', 
+        'checked': False
+      })
+      
+    return user
 
 
 def authUser(userName, password) :
     # your code here
+    output = False
+    
+    for user in authUser :
+      if authUser == user['user'] and user['checked'] == False :
```

Comment:
I don't know what `authUser` is supposed to be. It's the name of the function as far as I can tell. I think you mean

```python
    for user in users :
        if user['userName'] == userName and ...
```